package.json

```diff
@@ -53,6 +54,11 @@
     "truffle": "4.1.11"
   },
   "dependencies": {
-    "openzeppelin-solidity": "1.10.0"
+    "@linniaprotocol/linnia-js": "^0.3.0",
```

Comment:
This looks like it will not use the SC in this repo.  We should make sure that it is using this repos smart contracts not the contracts in NPM.

Reply:
Linnia is used for encryption and when getting contracts the current hub address is passed in as an option